protos/external.proto

```diff
@@ -39,6 +40,13 @@ service External {
   // Resume a pipeline; noop if pipeline is not paused
   rpc ResumePipeline(ResumePipelineRequest) returns (protos.StandardResponse);
 
+  rpc CreateNotification(CreateNotificationRequest) returns (protos.StandardResponse);
+  rpc UpdateNotification(UpdateNotificationRequest) returns (protos.StandardResponse);
+  rpc DeleteNotification(DeleteNotificationRequest) returns (protos.StandardResponse);
+  rpc GetNotifications(GetNotificationsRequest) returns (GetNotificationsResponse);
+  rpc GetNotification(GetNotificationRequest) returns (GetNotificationResponse);
+  rpc AttachNotification(AttachNotificationRequest) returns (protos.StandardResponse);
```

Comment:
Need to be able to Detach a notification.

Reply:
Fixed